ONSdigital/rm-action-service
# Social sample file pilot changes

### NOTE: This will need to be merged at the same time as the other java service sample file pilot change branches

# Motivation and Context

The format of social sample file is changing for the LMS pilot, these changes are needed to pass the new address format on action requests

# What has changed



* Changed the address format in the sample attributes decorator to use new column headers
* Set survey abbreviation on the action request
* Update tests

# How to test?



* Run acceptance tests
* Load a new format social sample file locally and execute a collection exercise with it, print files should still be generated with address information


# Links



https://trello.com/c/3PK5xIDM/238-sample-file-changes-for-operational-test-8
https://github.com/ONSdigital/rm-actionsvc-api/pull/28
https://github.com/ONSdigital/rm-samplesvc-api/pull/17
https://github.com/ONSdigital/rm-sample-service/pull/72
https://github.com/ONSdigital/rm-actionexporter-service/pull/48
https://github.com/ONSdigital/rasrm-acceptance-tests/pull/166
